lib/gemstash/cli.rb

```diff
@@ -66,5 +67,20 @@ def status
     def stop
       Gemstash::CLI::Stop.new(self).run
     end
+
+    desc "preload", "Preloads all the gems in your gemstash server from the default upstream"
+    method_option :latest, :type => :boolean, :default => false, :desc =>
+      "Only fetch the latest specs"
+    method_option :server_url, :type => :string, :default => "http://localhost:9292", :desc =>
+      "Gemstash server url"
```

Comment:
If I'm reading the code right, it looks like this is the upstream URL, not the Gemstash URL. If that's the case, that seems like it conflicts with the task description discussing the `default upstream`. If this is the upstream, I think it should probably be `nil` by default, so the task description will be correctly pulling from the `default upstream` when this is `nil` (which may require some tweaks to the code).